### ISSUE TITLE
segmentation results and iteration number setting

### ISSUE THREAD

**Issue by lemondan** [isht7/pytorch-deeplab-resnet#10]
Why do you use 20000 iteration size with only batch_size=1? In this way, each training image is only passed for 2 times to get the final model. 

Another question is that I have run  evalpyt2.py and your trained VOC12_scenes_16000.pth model to test. But I only got a validation performance of 66.7%. Do you know why it is so low? Thanks!

**Comment by isht7** [comment]
for the first question, it is done originally this way in deeplab, and therefore we also did it this way.
second, you should get 72.40% if you evaluate the downloaded `VOC12_scenes_16000.pth` with `evalpyt2.py`.

**Comment by lemondan** [comment]
In order to replicate 72.40, have you set all 255 labels into 0? And which annotations for validation set are used, SBD dataset or voc dataset?

**Comment by isht7** [comment]
All results are for VOC. SBD has not been used anywhere in this repo. I will add a point regarding this in the results section.

**Comment by isht7** [comment]
The result(72.40 mean IOU on VOC) is without merging the label 255 with 0. The merging was only done during training. If you look [here](https://github.com/isht7/pytorch-deeplab-resnet/blob/master/evalpyt2.py#L101), that line is commented out.

**Comment by lemondan** [comment]
Thanks! solved!

**Comment by isht7** [comment]
Closing this.
